### DISCUSSION TITLE
How to guide / Walkthrough on how to use this for PS4?

### DISCUSSION THREAD

**Discussion by DrNicket** [zencq/NomNom#53] [Q&A] [answered]
Is there a step by step anywhere for how to use this for PS4 saves?

**Comment by cengelha** [comment]
No, and I can't provide one myself as I don't own a PS4. But in the end, you should have a `memory.dat` file in all cases which contains all your saves at once. Select the path where you saved that file and NomNom will automatically detect and load it.

**Comment by cengelha** [comment, marked as answer]
@DrNicket created a very detailed step-by-step guide for this: [Step-by-Step for Playstation saves of No Man's Sky](https://docs.google.com/document/d/1QoD2-PNlX-HeR5K1zuPGLMLBcX4_wknkhzc43-9bEq4/edit?usp=sharing)
